modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -30,6 +31,9 @@ def initialize
           'BLANK_PASSWORDS' => true
         }
     )
+    register_options([
+      OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', false]),
```

Comment:
```suggestion
      OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', true]),
```

I believe this was the default we were after